osbuild/buildroot.py

```diff
@@ -137,7 +138,7 @@ def register_api(self, api: "BaseAPI"):
         if self._exitstack:
             self._exitstack.enter_context(api)
 
-    def run(self, argv, monitor, binds=None, readonly_binds=None):
+    def run(self, argv, monitor, timeout, binds=None, readonly_binds=None):
```

Comment:
Alternatively you could make it an instance/class variable with a default of `None`; or make it a proper keyword arg with the default `None`, then you dont have to modify all the existing usage, including the tests.

Reply:
Sure!

Which do you prefer and ill get on it?
Making it a keyword arg like `binds` sounds like the lightest lift, but is it best for the project?

Reply:
I went ahead and made the `stage_timeout` a keyword arg in buildroot.py. 
Take a look and if you prefer I can change it to a class var.